src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -169,7 +178,8 @@ public static IEnumerable<object[]> InvalidNtlmNegotiateAuthentication_TestData(
             yield return new object[] { "abcd", HttpStatusCode.BadRequest };
         }
 
-        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // [PlatformSpecific(TestPlatforms.Windows, "Managed impl doesn't support NTLM")]
```

Comment:
I suppose that the ones where we have a comment of an active issue I shouldn't bring back? like:
```c#
[ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // [ActiveIssue(20098, TestPlatforms.Unix)]
```